tasks/create_vnc_user.yml

```diff
@@ -0,0 +1,73 @@
+---
+#
+# Create and configure the vnc user
+#
+- name: Create the VNC user
+  ansible.builtin.user:
+    name: "{{ username }}"
+    shell: /bin/bash
+
+- name: Create .vnc directory for VNC user
+  ansible.builtin.file:
+    path: /home/{{ username }}/.vnc
+    state: directory
+    mode: 0755
+    owner: "{{ username }}"
+    group: "{{ username }}"
+
+- name: Set VNC password for VNC user
+  ansible.builtin.shell: |
+    set -o pipefail
+    echo {{ password }} | vncpasswd -f > /home/{{ username }}/.vnc/passwd
+  args:
+    executable: /bin/bash
+    creates: /home/{{ username }}/.vnc/passwd
+
+- name: Set correct permissions for VNC passwd file
+  ansible.builtin.file:
+    path: /home/{{ username }}/.vnc/passwd
+    owner: "{{ username }}"
+    group: "{{ username }}"
+    mode: 0600
+
+#
+# Install an SSH keypair for the VNC user.  This will enable Guacamole
+# to perform file transfers to/from this instance.
+#
+# Note that Guacamole (guacd, as of v.1.1.0) currently only supports
+# RSA keys.
```

Comment:
Did you happen to check if Guacamole supports non-RSA keys yet?  Not a big deal for this PR, but it would be good to know.  I don't recall if it was documented anywhere- I may have just discovered this by trial and error.

Reply:
Looking at [the source code](https://github.com/apache/guacamole-server/search?q=rsa) it appears to only support RSA and DSA.

Reply:
Thanks for checking on that!